### PR TITLE
Update the Python-Wechaty

### PR DESCRIPTION
**Pull request by shraddhavp**
This PR links to #890 and this PR contains the updated document of the Python-Wechaty.

### CODE REVIEW

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
Pass the CI and we are good to go ig

File: docusaurus/docs/polyglot/python/overview.md [hunk 1]
```diff
@@ -6,12 +6,22 @@ sidebar_label: Python
...
-- Getting Started Template Repo <https://github.com/wechaty/python-wechaty-getting-started>
+Python-wechaty is a Python programming language client derived from the Wechaty ecosystem, allowing developers to connect to various instant messaging software platforms with a small amount of code.Python-wechaty has been committed to improve code robustness, add community out-of-the-box tools, and improve the software development documentation.
+
+## Features of Python-Wechaty
+
+* Send and receive messages.
+* Accept friendship from contact.
+* Creation of room, member invitation.
+* Can also use the Wechaty-plugin system.
+* This has a  powerful sample code library for robots.
```
Comment (proudofsimin) [inline docusaurus/docs/polyglot/python/overview.md]: When we use bulleted list, the sentence structures should be paralleled, so maybe consider changing to "Provide powerful sample code library for robots."

File: docusaurus/docs/polyglot/python/overview.md [hunk 2]
```diff
@@ -6,12 +6,22 @@ sidebar_label: Python
...
 
-- GitHub Repo <https://github.com/wechaty/python-wechaty>
-- Getting Started Template Repo <https://github.com/wechaty/python-wechaty-getting-started>
+Python-wechaty is a Python programming language client derived from the Wechaty ecosystem, allowing developers to connect to various instant messaging software platforms with a small amount of code.Python-wechaty has been committed to improve code robustness, add community out-of-the-box tools, and improve the software development documentation.
+
+## Features of Python-Wechaty
+
+* Send and receive messages.
+* Accept friendship from contact.
+* Creation of room, member invitation.
```
Comment (proudofsimin) [inline docusaurus/docs/polyglot/python/overview.md]: When we use bulleted list, the sentence structures should be paralleled, so maybe " Create rooms and invite members."

File: docusaurus/docs/polyglot/python/overview.md [hunk 3]
```diff
@@ -6,12 +6,22 @@ sidebar_label: Python
...
+* Can also use the Wechaty-plugin system.
```
Comment (proudofsimin) [inline docusaurus/docs/polyglot/python/overview.md]: Support Wechaty-plugin system.

File: docusaurus/docs/polyglot/python/overview.md [hunk 4]
```diff
@@ -6,12 +6,22 @@ sidebar_label: Python
...
+* This has a  powerful sample code library for robots.
+* It is very easy to implement.
```
Comment (proudofsimin) [inline docusaurus/docs/polyglot/python/overview.md]: Easy to implement

File: docusaurus/docs/polyglot/python/overview.md [hunk 5]
```diff
@@ -20,18 +30,28 @@ make install
 make bot
 ```
 
+## Translation of TypeScript to Python
+
+As the basic ecosystem of wechaty is in TypeScript we only need to translate it to Python .There is a hundred (100) lines of code in  class named `Image` in charge of downloading the WeChat image to different sizes.
```
Comment (proudofsimin) [inline docusaurus/docs/polyglot/python/overview.md]: Be careful of the spaces:
As the basic ecosystem of Wechaty is in TypeScript, we only need to translate it to Python. There is a hundred (100) lines of code in class named `Image` in charge of downloading the WeChat image to different sizes.

File: docusaurus/docs/polyglot/python/overview.md [hunk 6]
```diff
@@ -20,18 +30,28 @@ make install
...
+Below are the links to `Image` class for both TypeScript and Python for better understanding of the code and the translation.
+
+* TypeScript: <https://github.com/wechaty/wechaty/blob/master/src/user/image.ts>
+* Python: <https://github.com/wechaty/python-wechaty/blob/master/src/wechaty/user/image.py>
+
 ## History
 
-- [Python Wechaty Beta Released: A Simple Bot, wj-Mcat, Jun 17, 2020](https://wechaty.js.org/2020/06/17/python-wechaty-beta-released/)
+For more information on the Python-Wechaty releases and its history the below link can be visited.
```
Comment (proudofsimin) [inline docusaurus/docs/polyglot/python/overview.md]: For more information on the Python-Wechaty releases and history, visit the link below:

File: docusaurus/docs/polyglot/python/overview.md [hunk 7]
```diff
@@ -6,12 +6,22 @@ sidebar_label: Python
...
+## Features of Python-Wechaty
+
+* Send and receive messages.
+* Accept friendship from contact.
+* Creation of room, member invitation.
+* Can also use the Wechaty-plugin system.
+* This has a  powerful sample code library for robots.
+* It is very easy to implement.
+
+You can find more information on Python-Wechaty [here](https://github.com/wechaty/python-wechaty-getting-started).The [General Architecture Diagram](https://wechaty.js.org/docs/polyglot/diy/) illustrates on how the Python-Wechaty can be implemented on the already existing TypeScript Wechaty ecosystem.
```
Comment (proudofsimin) [inline docusaurus/docs/polyglot/python/overview.md]: You can find more information on [python-wechaty-getting-started](https://github.com/wechaty/python-wechaty-getting-started). The [General Architecture Diagram](https://wechaty.js.org/docs/polyglot/diy/) illustrates on how the Python-Wechaty can be implemented on the ~already~ existing TypeScript Wechaty ecosystem.

**Review by proudofsimin — CHANGES_REQUESTED** [review]

**Review by proudofsimin — APPROVED** [review]
Thanks :)

**Review by huan — APPROVED** [review]
LGTM.

Please read and follow #1257, and feel free to help yourself to merge it by following the merge workflow.

Thank you very much!

**Review by wj-Mcat — APPROVED** [review]
LGTM

**Pull request merged**